packages/api-electron/src/preload/app.ts

```diff
@@ -1,5 +1,7 @@
-const ready = () => Promise.resolve();
+class app implements ssf.App {
```

Comment:
Why change this to a class? Seems like a unrelated change and I'm not sure what the benefit is? If it does make sense, the should `app` be capitalised to `App` for consistency?